plugins/cryptolib/cryptolib.cpp

```diff
@@ -428,6 +428,10 @@ void loadLFS(const char * lfs, IUserDescriptor * user, StringBuffer &sb)
         }
 
         IDistributedFilePart &part = df->queryPart(0);
+        if (!part)
```

Comment:
@RussWhitehead - can you change it to test df->numParts()==0 instead?

It's probably illegal (or undefined) testing a reference for null.
NB: queryPart should probably thrown an exception rather than try and return a dodgy null reference, but don't change as part of this PR.